boards/arm/atsamd20_xpro/atsamd20_xpro.dts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018 Sean Nyekjaer
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <atmel/samd20.dtsi>
+
+/ {
+	model = "SAM D20 Xplained Pro";
+	compatible = "atsamd20,xpro", "atmel,samd20j18", "atmel,samd20";
+
+	chosen {
+		zephyr,console = &sercom3;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&sercom0 {
+	status = "ok";
+	compatible = "atmel,sam0-spi";
+};
+
+&sercom2 {
+	status = "ok";
+	compatible = "atmel,sam-i2c-twi";
```

Comment:
Huh, is this correct?  I thought the I2C port was also a SERCOM mode...

Reply:
are we still missing an i2c driver for the sam0?

Reply:
@sknsean Corrrect. There is no in-tree TWI driver for the sam0 series yet.

Reply:
Great one more thing on the TODO list :-)